tutorial/developing_packages/editable_packages/hello/conanfile.txt

```diff
@@ -0,0 +1,9 @@
+[requires]
```

Comment:
I am not sure I love using a ``conanfile.txt`` here at this point, but sounds a ``conanfile.py`` would be a bit more homogeneous.